src/main/resources/webapp/interface/post.ts

```diff
@@ -77,7 +93,7 @@ function isUnauthorized({
 			status: 401 // Unauthorized
 		};
 	}
-	const apiKeyNode = explorerRepoReadConnection.get(matchingApiKeys.hits[0].id);
+	const apiKeyNode = explorerRepoReadConnection.get<ApiKeyNode>(matchingApiKeys.hits[0].id);
```

Comment:
A different casting syntax is used in other files. 

`value as type` instead of `value<type>` 
For consistency please pick one and stick to it.

Reply:
No this is casting type into the function. Not casting the result. Without that, typescript will give an error